vllm/executor/ray_utils.py

```diff
@@ -229,12 +229,17 @@ def initialize_ray_cluster(
             the default Ray cluster address.
     """
     assert_ray_available()
-
     # Connect to a ray cluster.
     if is_hip() or is_xpu():
-        ray.init(address=ray_address,
-                 ignore_reinit_error=True,
-                 num_gpus=parallel_config.world_size)
+        # Try to connect existing ray instance and create a new one if not found
+        try:
+            ray.init('auto')
+        except ray.exceptions.ConnectionError:
+            logger.warning("No existing RAY instance detected. " +
+                           "A new instance will be launched with current node resources.")
```

Comment:
@yma11 it looks like you need to run `format.sh` and you'll see that it's generating an error that this line (and one other) is too long. That's why CI is failing.

Reply:
Thanks for comments. I will update this PR soon. Just working on fix some issues in my env to run `format.sh`.